### PR TITLE
[Task] Add MMT and MMT_MI (Multiple Image) Task

### PR DESCRIPTION
**Pull request by ngquangtrung57**
This pull request adds two new tasks: MMT and MMT_MI (Multiple Image) from [here](https://mmt-bench.github.io/).

MMT: This task processes a single image per example.
MMT_MI: This task processes multiple images per example.

For each task, we have two evaluation splits:

Validation (val): This split returns overall accuracy and sub-category accuracy.
Test: This split generates a submission file for submission on [EvalAI](https://eval.ai/web/challenges/challenge-page/2328/overview).

### CODE REVIEW

**Comment by Luodian** [comment]
Hi please see the lint issue and fix it first, thanks!

File: lmms_eval/tasks/mmt/utils_mi.py
Comment (kcz358) [inline lmms_eval/tasks/mmt/utils_mi.py:1]: Hi, may I ask is this file duplicated? Check using file difference checker and does not seems to have any difference except the format of the doc to visual. Is it possible that you can put the `doc_to_visual` of mi into the utils and specify using the yaml config?

Comment (ngquangtrung57) [reply lmms_eval/tasks/mmt/utils_mi.py:1]: Yes, the only different between them is the doc_to_visual. I will move the doc_to_visual of mi to the utils and remove the utils mi file. Thanks!

**Review by kcz358 — APPROVED** [review]
Looks all the codes are quite good to me. Can you share a screenshot of the evaluation result and submission result? Then I think this PR would be good to merge. Thank you!

**Comment by ngquangtrung57** [comment]
@kcz358 
[mmt_test_submission.json](https://github.com/user-attachments/files/17084040/mmt_test_submission.json)
[mmt_val.json](https://github.com/user-attachments/files/17084041/mmt_val.json)
[results.json](https://github.com/user-attachments/files/17084043/results.json)
Hi, thanks for your review. These are the results of the evaluation using llava onevision 0.5B.
There still have the mmt_test file but I can't upload it here.

**Pull request merged**